### PR TITLE
Dl 1 final submission

### PR DESCRIPTION
**Pull request by PriyanshuPansari**
First Submission 
Got 97.81% accuracy on train set
Work to be done-
Create validation and test set and test the model on it
Hyperparameter tuning
Documentation

### CODE REVIEW

**Comment by PriyanshuPansari** [comment]
Getting 96.10714285714286 test accuracy, Is it good enough or does the model needs more tuning?
